apps/expo/app/_layout.tsx

```diff
@@ -57,6 +75,23 @@ export default function HomeLayout() {
     void startAgent()
   }, [])
 
+  // Setup mediation
+  useEffect(() => {
+    if (!agent) return
+    if (!hasInternetConnection || isMediationConfigured) return
+
+    void hasMediationConfigured(agent).then(async (mediationConfigured) => {
```

Comment:
Do you think there is a chance of this triggered twice before the async function is actually resolved? Or will that be handled inside afj?

Reply:
Yeah good point. I can add a state param that tracks whether the state is currently loading